corehq/apps/toggle_ui/shortcuts.py

```diff
@@ -1,7 +1,7 @@
 from couchdbkit import ResourceNotFound
 from django.conf import settings
 
-from .models import Toggle
+from corehq.apps.toggle_ui.models import Toggle
```

Comment:
Why did this need to change?

Reply:
It didn't, I'm just not fond of relative imports. But I think I'm in a minority there. Updated.